pkgs/development/python-modules/mando/default.nix

```diff
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "mando";
+  version = "0.6.4";
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "rubik";
+    repo = "mando";
+
+    # this rev contains a fix for newer versions of python (3.8+), but is still tagged 0.6.4
+    rev = "460c2ed296f04530a84d39c1e547b143f11b7580";
+    sha256 = "sha256-Ge73kNrE5PeiFKy1lxOC3OS/0b5HWfGz95ZuLcMJCkI=";
+  };
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+  disabledTests = [
```

Comment:
Please add a note why these tests have to be disabled

Reply:
good call! Done